src/fleet/azext_fleet/_help.py

```diff
@@ -340,3 +340,48 @@
     short-summary: Wait for a update strategy resource to reach a desired state.
     long-summary: If an operation on an update strategy was interrupted or was started with `--no-wait`, use this command to wait for it to complete.
 """
+
+helps['fleet autoupgradeprofile'] = """
+    type: group
+    short-summary: Commands to manage auto upgrade profiles.
+"""
+
+helps['fleet autoupgradeprofile create'] = """
+    type: command
+    short-summary: Creates or updates an auto upgrade profile.
+    examples:
+        - name: Create an auto upgrade profile with channel 'Rapid' and disabled by default.
+          text: az fleet autoupgradeprofile create -g MyFleetResourceGroup -l MyLocation -f MyFleetName -n MyAutoUpgradeProfileName -c Rapid --disabled
+        - name: Create an auto upgrade profile with channel 'NodeImage'.
+          text: az fleet autoupgradeprofile create -g MyFleetResourceGroup -l MyLocation -f MyFleetName -n MyAutoUpgradeProfileName -c NodeImage --node-image-selection-type Latest
```

Comment:
fix the example to match the service validation on node-image-selection-type (cannot be set for nodeimage channel)